### ISSUE TITLE
Get frontend accessibility to AA

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/PlaceCal#2160]
## Description
Run down accessibility checklist to make sure we're doing all the right things. Obviously we will be accessibility first as we build but it's worthwhile to do an audit after we have built a few pages.

## Acceptance Criteria

Must do:
 - [x] Run something like lighthouse (https://developer.chrome.com/docs/lighthouse/overview/)
 - [x] Use a WAVE tool to evaluate each page type https://wave.webaim.org/
 - [ ] Check all links for meaningful & unique link text
 - [x] Check all headings are correct for their semantic intent

Consider also:

- [x] run through site with keyboard
- [ ] run through site with screen reader
- [ ] Run down https://www.w3.org/WAI/WCAG21/quickref and ticket anything that does not meet AA
- [x] Starling bank has a nice statement https://www.starlingbank.com/website-accessibility. We ought to be doing all the things listed there as "What we are doing"
 - [x] Check all form inputs for labels
 - [x] Check all meaningful images for alt text

## Implementation notes & questions

This is a nice reference - https://www.understandingaccessibility.com/

## Implementation plan

_To be written by the developer_

**Comment by aaaaargZombies** [comment]
## wave tool

### homepage
- no language set in `head`
- burger menu is a link `a` element that goes nowhere should be a button.

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/b92f6284-f73f-4149-8712-1c7cf771e646)
- button text is not sufficient contrast
- button has no visible focus state
- logo and "find out more" same / redundant link

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/bfd59975-a0c2-4901-bff5-82799a61b9ef)
- insufficient contrast red/brown
- button has no visible focus state  (should be fixed at component level, I will stop listing this)

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/8930e3d0-b516-4a2d-9721-6d614818f2c3)
- "Created by" / "sponsored by" are `h5` : skipped heading levels

**Comment by aaaaargZombies** [comment]
## wave tool

I'll not record issues from footer/header as they will be repeats from the homepage

### Our Story
![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/ea6cef8b-0230-4c35-9986-63698d8ee76d)
![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/2ab9f9b3-518b-4e66-b90d-48fae6403af6)
![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/9e314b28-4803-4e51-b307-60ec065fcb93)
![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/7dae1ed3-6fb0-4e23-84f9-d2390b36d2d0)
![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/6ead0950-a8a0-45c4-b38b-594cfe596aaa)
![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/41186152-10d9-4c03-8e8c-07793c7b8f71)
![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/22354f42-a752-4deb-86bb-9e76603e4fdd)
- all have missing alt-text

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/02ff0647-5e15-46c4-b672-38aadf20d3a4)
- cream on green very low contrast

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/1bcc17a0-184b-436a-9f94-6bbf6691a939)
- cream on pink also poor contrast

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/fe9e7650-4a72-4a31-8b80-fd5f8fab3126)
- brown on pink would need to be "large text" to meet minimal contrast thresholds

**Comment by aaaaargZombies** [comment]
## wave tool
### Find your PlaceCal

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/feb0bd19-6dae-4a78-8a75-e24c214a1266)
- cream on green is low contrast

**Comment by aaaaargZombies** [comment]
## wave tool
### Get in touch

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/89a567e7-fca6-4e30-80e6-238e33cbb563)
- this flagged a whopping 17 contrast errors and that doesn't include user input text which are maybe the worst.

**Comment by aaaaargZombies** [comment]
## wave tool
### https://manchester.placecal.org/

same head lang and link for burger menu issues as all other pages

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/dd17f1f4-2ebf-4d01-a4fb-e3490f1aac3b)
- not getting flagged because the wave tool can't pick up the background color when it's an image but this is not meeting contrast thresholds

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/05137aab-56df-4319-bffd-68dd273fe95f)
- footer logo missing alt-text

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/71c077c3-fbad-4b0d-a94f-4c96fe76ff13)
- contrast again on buttons

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/b692b2e7-cdc8-4748-8899-da2db40ab533)
- these are duplicate links, wave tool doesn't catch it because it's redirect

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/336561aa-a087-4ada-b378-b5ac1104a46f)
- super bad contrast in footer

**Comment by aaaaargZombies** [comment]
## wave tool
### https://manchester.placecal.org/events

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/a0b363c9-0cf2-4f72-93b7-a433c163bce9)
- fails contrast

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/cffedffa-f47c-44f2-9fd5-0ef20411bf2a)
- fails contrast

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/b59c7cd8-c7af-4ca8-a5ad-9f41ace440ad)
- fails contrast
- skipped header level `h4`

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/5c076288-aaa0-4904-8f44-058c926c2bfe)
- checkboxes / radio buttons not grouped into fieldset

**Comment by aaaaargZombies** [comment]
## wave tool
### https://manchester.placecal.org/partners

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/14cd029d-79cd-4bbf-8f98-2cfc00b79dff)
![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/9872338c-b113-48bb-b0ae-8e782e9fbf7a)
- none of the labels / inputs are linked

![image](https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/87a8dac7-8e1a-44a8-9d5a-f0a6cf08f7a7)
- skipped header level `h3`

**Comment by aaaaargZombies** [comment]
## wave tool
### partner https://manchester.placecal.org/partners/age-friendly-hulme-moss-side

- contrast errors that are covered above
- getting a few warnings about heading hierarchy /  skipped levels and Firefox reader mode is chopping out all the events.  Probably needs a review to draw out what we think it should do as a structured document, some of these things might just be styled text.

**Comment by aaaaargZombies** [comment]
## wave tool
### event https://manchester.placecal.org/events/330983

- contrast and header issues but all duplicates of other pages.

**Comment by aaaaargZombies** [comment]
I did run lighthouse but it hasn't picked up anything new

**Comment by aaaaargZombies** [comment]
https://github.com/geeksforsocialchange/PlaceCal/assets/6824891/8f9176b9-e98c-4792-ba4d-13a5051dde6e

Noticed that the event titles are links but lack the same link styling used elsewhere they look a lot like headings, would be nice if we had some sort of state change too on hover except the lil hand.
